src/pages/[id]/index.js

```diff
@@ -50,11 +47,26 @@ const MemberProfile = ({ imageLink, user, contributions, errorMessage }) => {
     fillActiveTasksArray();
   }, [fillActiveTasksArray]);
 
+  if (errorMessage) {
+    return <NotFound errorMsg={errorMessage} />;
+  }
+
+  if (!isSuperUser) {
+    if (!apiCalledToVerifySuperUser) {
+      makeApiCallToVerifySuperUser();
```

Comment:
Since makeApiCallToVerifySuperUser is async shouldn't there be an await here?

Reply:
I tried adding await here like this
```
    if (!apiCalledToVerifySuperUser) {
      (async () => await makeApiCallToVerifySuperUser())();
    }
``` 
but I am getting an error in the git log when committing as follows 
```
error  Redundant use of `await` on a return value  no-return-await
```